README.md

```diff
@@ -19,6 +19,12 @@ Your app should now be running on [localhost:5000](http://localhost:5000/).
 
 ## Deploying to Heroku
 
+Using resources for this example app counts towards your usage. [Scale your dynos to 0](https://devcenter.heroku.com/articles/scaling#scaling-from-the-dashboard) and [delete your database](https://devcenter.heroku.com/articles/heroku-postgresql#removing-the-add-on) as soon as you are done experimenting to control costs.
```

Comment:
Scaling to 0 and deleting your db is fine, but maybe we should suggest deleting the app as an alternative? This app won't be useful once it has no dynos or DB. It's served it's purpose already, too: teaching someone to deploy Node.js on Heroku.

Reply:
For myself, I'd rather delete the apps so it doesn't pollute my dashboard and I won't get stack deprecation emails about it either.

Reply:
agreed. copy changes approved and updated.